dispatch_examples/greeter/Cargo.toml

```diff
@@ -7,13 +7,13 @@ edition = "2021"
 frc42_dispatch = { path = "../../frc42_dispatch" }
 fvm_ipld_blockstore = "0.1.1"
 fvm_ipld_encoding = "=0.3.3"
-fvm_sdk =  "=3.0.0-alpha.24"
-fvm_shared = { version = "=3.0.0-alpha.20" }
+fvm_sdk = "=3.0.0"
```

Comment:
I'd change this to `~3.0` so we get patch releases without having to make updates here.

(same below, really).